core/server/utils/image-size-from-url.js

```diff
@@ -79,9 +79,16 @@ module.exports.getImageSizeFromUrl = function getImageSizeFromUrl(imagePath) {
                             context: imagePath
                         }));
                     }
+                } else if (res.statusCode === 404) {
```

Comment:
I don't know what's happening 😱  you're totally right.

I swear I _was_ getting 2 errors for every image size error in my stdout, and this was reducing it down to 1. Must've got my branches mixed up or something.

Either way I've added a new commit which now handles the different error types differently.